modules/knative-serving-CR-system-deployments.adoc

```diff
@@ -1,14 +1,12 @@
 // Module included in the following assemblies:
 //
-// * /serverless/admin_guide/knative-serving-CR-config.adoc
+// * serverless/admin_guide/serverless-configuration.adoc
 
 :_content-type: REFERENCE
 [id="knative-serving-CR-system-deployments_{context}"]
 = Overriding system deployment configurations
 
-You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resource (CR).
-
-Currently, overriding default configuration settings for the `replicas`, `labels`, `annotations`, and `nodeSelector` fields are supported.
+You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resource (CR). Currently, overriding default configuration settings for the `replicas`, `labels`, `annotations`, and `nodeSelector` fields are supported.
```

Comment:
Change to: "...fields is supported.

Reply:
rewriting to read better as: "Currently, overriding default configuration settings is supported for the `replicas`, `labels`, `annotations`, and `nodeSelector` fields."